src/server/channel_store.cc

```diff
@@ -23,6 +25,32 @@ bool Matches(string_view pattern, string_view channel) {
   return stringmatchlen(pattern.data(), pattern.size(), channel.data(), channel.size(), 0) == 1;
 }
 
+// Build functor for sending messages to connection
+auto BuildSender(string_view channel, facade::ArgRange messages) {
+  absl::FixedArray<string_view, 1> views(messages.Size());
```

Comment:
what's wrong with `std::array` ? Obviously I don't care if you use `absl::FixedArray` just asking if there is actually a difference

Reply:
absl::FixedArray is dynamically sized, the `1` is just the inline size